index.js

```diff
@@ -3,15 +3,20 @@ const async = require('async');
 const Logr = require('logr');
 const path = require('path');
 const fs = require('fs');
+const defaults = require('lodash.defaults');
 const bytesize = require('bytesize');
 
+let DefaultOptions = {
+  description: 'Default ClientKitTask Description',
+};
+
 class ClientKitTask {
   constructor(name, options, runner) {
-    options = options || {};
     this.name = name;
-    this.options = options;
-    this.runner = runner;
-    this.log = new Logr({
+    this.options = defaults(options, DefaultOptions);
```

Comment:
Defaults should be on the class. Idea is that specific tasks will have their own options and the task needs to be able to set it